### PR TITLE
Marketplace: Lay out the HCI for Shopper Purchasing Products

### PR DESCRIPTION
**Pull request by zspencer**
https://github.com/zinc-collective/convene/issues/831

From: https://github.com/zinc-collective/convene/pull/1057#discussion_r1082153560

### CODE REVIEW

File: app/furniture/marketplace/README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Marketplace
 
+## Human/Computer/Human Interaction Design
+
+### Shopper Purchases Products
+
+<!-- it would be cool if someone could Mermaid this -->
```
Comment (KellyAH) [inline app/furniture/marketplace/README.md:7]: wow [Mermaid](https://mermaid.js.org/) looks sick/cool! **- TODO @KellyAH  🧜‍♀️ it!**

**Review by anaulin — APPROVED** [review]

File: app/furniture/marketplace/README.md (hunk 1 above)
Comment (zspencer) [reply app/furniture/marketplace/README.md:7]: It *is* cool! I am just not fluent in it (yet!)

Comment (KellyAH) [reply app/furniture/marketplace/README.md:7]: was there a specific type of diagram you hoping for? Sequence, flow, state, user journey or something else? see [Sample Diagrams](https://mermaid.live/edit#pako:eNplULsKwzAM_BWhOUspXbw20MlT1iwiFo3bxgqOTAih_17nBS3VJO5Od5JmbMQxGnxIioGnOkAu9fpisBOMEp8-3MHRzgzcqJcANwGVld5wAEtPBmUycDFg-YCzMPWDko-DgfM3U27zBk4LWsCV9C-jle7byckYDq-flMrrSu4RWGDHsSPv8mXzIqpRW85eaHLrKK-NdXhnHSWVagoNGo2JC0y9I-XS0z1Sd4DsvEq026fWh70_H4Fhow)

Comment (KellyAH) [reply app/furniture/marketplace/README.md:7]: Here's a rough draft diagram @zspencer 
Is this okay to digitize w/ 🧜‍♀️  or should I "go back to the drawing board"? 😉 

shopping checkout flow draft

Comment (KellyAH) [reply app/furniture/marketplace/README.md:7]: Coded in 🧜‍♀️  in https://github.com/zinc-collective/convene/pull/1064

**Pull request merged**